web/packages/teleport/src/AuthConnectors/AuthConnectorEditor/GitHubConnectorEditor.story.tsx

```diff
@@ -0,0 +1,122 @@
+/**
+ * Teleport
+ * Copyright (C) 2024  Gravitational, Inc.
```

Comment:
```suggestion
 * Copyright (C) 2025  Gravitational, Inc.
```

Here and below in other new files please